### ISSUE TITLE
Ubuntu 14.04 Fails to build with bento/packer

### ISSUE THREAD

**Issue by timvisher** [chef/bento#477]
I was trying to build an ubuntu 14.04 box just now and can't seem to succeed. It looks like something goes wrong in the initial boot process and then it gets stuck in the language selection screen.

| Key | Value |
| ---------------- | ------------------ |
| Host OS | Mac 10.11 (15A284) |
| Guest OS | `bin/bento build --only=virtualbox-iso ubuntu-14.04-amd64` @ `89214273c0c7ad61dd41a6df3ba37bbbc4189fe7` |
| arch | amd64 iiuc (`3 GHz Intel Core i7`) |
| packer | `0.8.6` |
| virtualbox |  `5.0.6r103037` |

Thoughts?

![bento ubunut 14 04 bug](https://cloud.githubusercontent.com/assets/34676/10561995/f24ba986-7512-11e5-9eab-c5ee5dbf3488.gif)

**Comment by timvisher** [comment]
It looks like _roughly_ the same thing is happening on CentOS 7.1

**Comment by rickard-von-essen** [comment]
See my comment on https://github.com/mitchellh/packer/issues/2156, is is usually a problem with your host being to slow/under heavy load. Try decreasing the load or  you run in a nested VM assign more resources.

**Comment by cheeseplus** [comment]
@timvisher as @rickard-von-essen mentions it's likely a load issue - I also see a bunch of random key presses (tttttttttt) in the output which is likely causing problems.

Also when reporting these issues _please_ be specific to the OS, arch, packer version, as well as the virtualbox (or specific provider) versions.

**Comment by timvisher** [comment]
@cheeseplus Added the versioning information to the description here (did I miss the contributing to this repo link when opening the issue?).

As far as reducing the load, my load is consistently `~1.5-2` which I was trained to react to as ridiculously high, but doesn't seem to negatively affect performance for any other applications. I tried quitting essentially everything else and running the build and the same thing happened.

**Comment by cheeseplus** [comment]
Didn't miss anything, but there should be an ISSUES.md added with some requirements as most reproing issues with virtualization/packer is already tricky.

The only things that stand out to me at the moment are 1) the errant keystrokes I am noticing in the gif (right after the language screen and 2) 10.11 - we haven't upgraded the builders our of early adoption fears

**Comment by timvisher** [comment]
@cheeseplus 

> 1) the errant keystrokes I am noticing in the gif (right after the language screen

Yeah. I figured that was the boot command being typed before the vm was ready to receive it.

> 2) 10.11 - we haven't upgraded the builders our of early adoption fears

If you'd like I can serve as a guinea pig for 10.11 compat branches.

**Comment by cheeseplus** [comment]
The keypresses from that gif don't correspond to anything in the template which would just be the `boot_command` section for the respective builders.

We shouldn't really need to do much for 10.11 compat on this side unless things on the packer or hypervisor side break/change. That said I'm upgrading one of the builders to 10.11 to test things now that it's been a bit since release.

**Comment by cheeseplus** [comment]
Also seeing similarities in https://github.com/mitchellh/packer/issues/1796 - seems this may just be a side affect of an undue load with regard to packer.

**Comment by cheeseplus** [comment]
@timvisher I guess the real question is whether you can build _anything_ with Packer - while I totally believe this is an issue I'm not sure it's specific to the bento templates themselves. I've tried to repro on my most resource constrained builder (Mac Mini 2.5Ghz i5 / 4GB RAM / HDD) but thus far can't force the condition to occur

**Comment by timvisher** [comment]
So I lied, I suppose. I restarted my box and quit literally everything. This got my loadavg down to `~1.5`. I then opened Terminal.app (I generally use iTerm but I've heard that it might have issues with `notifyd` on `10.11`) and ran a `bin/bento build --only=virtualbox-iso ubuntu-14.04-amd64` and it worked. No double key presses or anything. One notable thing that happens is that the initial dialog box flickers quite a bit. It looks like Cancel is being pressed over and over again. Don't know if that points us in any direction.

During the build the loadavg spike to `~2.5` at times but never went over that. My loadavg tends to sit around `2`.
